src/overview.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Dart overview
+description: A short introduction to Dart
+---
+
+Dart is a client-optimized language for fast apps on any platform. Our goal is
```

Comment:
I wonder if it's time to reconsider out discussion about "client" as our label. As mentioned before I'd feel more comfortable with something like 'app-optimized'.

Reply:
Yeah, but "app-optimized" isn't specific enough. Server apps, cloud apps, client apps, etc. I'm resolving this conversation, but we can continue it offline to see if we can find something workable.